src/magic_uv/op/align_uv_cursor.py

```diff
@@ -151,6 +151,19 @@ def poll(cls, context):
             return True
         return _is_valid_context(context)
 
+    def draw(self, _):
+        layout = self.layout
+        column = layout.column()
+
+        row = column.row()
+        row.label(text="Align Method:")
+        row = column.row()
+        row.prop(self, "base", expand=True)
+        row = column.row()
+        row.label(text="Align Position:")
+        row = column.row()
+        grid = layout.grid_flow(columns=3, align=True)
+        grid.prop(self, "position", expand=True)
```

Comment:
@EugeneDudavkin 

Sorry, I send you the improper suggestion.
Could you apply this too?

```suggestion
        grid.prop(self, "position", expand=True)

    def execute(self, context):
```